lib/api/objectPut.js

```diff
@@ -119,7 +120,7 @@ function _storeIt(bucketName, objectKey, objMD, authInfo, canonicalID,
                                 newByteLength: size,
                                 oldByteLength: prevContentLen,
                             });
-                            return callback(null, contentMD5, prevContentLen);
```

Comment:
Since prevContentLen has been removed from both the callbacks, is it even necessary to calculate anymore (line 79)? Looks like it's just being logged in utapi